frontend/components/Borrow/BorrowToast.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { View, StyleSheet, Text } from 'react-native';
+import { Snackbar } from 'react-native-paper';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+
+function BorrowToast(props) {
```

Comment:
Let's object destructure props here. Since this is something that might be reused, I think it's better to make it clear to any one reading your code exactly which props your component expects.

This basically means change `function BorrowToast(props)` to `function BorrowToast({visible, onCancel})` and change the render appropriately.